openapi/components/schemas/Fees/Formulas/PercentageFormula.yaml

```
@@ -0,0 +1,32 @@
+type: object
+required:
+  - type
+  - currency
+  - bips
+properties:
+  type:
+    type: string
+    enum: ["percentage"]
+  currency:
+    $ref: ../../CurrencyCode.yaml
+  bips:
+    $ref: ../../Bips.yaml
+  round:
+    type: string
+    enum: ["regular", "up", "down"]
+    description: >
+      A mode how the rounding should work: floor, ceiling, or regular rounding:
+      - "up" (ceiling) example: $0.011 becomes $0.02, $0.019 becomes $0.02;
+      - "down" (floor) example: $0.011 becomes $0.01, $0.019 becomes $0.01;
+      - "regular" example: $0.011 becomes $0.01, $0.019 becomes $0.02, $0.015 becomes 0.02 too;
+
+      Defaults to "regular".
```

Looks excessive to repeat the schema in the description.
```suggestion

      Defaults to "regular".
```

---

It's work around for linter which requires punctuation at the end of description.

---

Hm, you could've just put the dot in the end of the list lines instead of `;`

---

This seems grammatically incorrect. Is it so important to you and block you to approve this PR?

---

This is not urgent to merge API, APIs are public, and why have a thing when it can be omitted. Not sure about grammatical rules which prohibit lists to end with a dot.